automatic/thunderbird/thunderbird.nuspec

```diff
@@ -19,11 +19,13 @@
 ## Package Parameters
 
 - `/l:LOCALE` - Install given Firefox locale. See the [official page](https://releases.mozilla.org/pub/thunderbird/releases/latest/README.txt) for a complete list of available locales.
+- `/NoRestart` - Do not attempt to restart the Thunderbird process
```

Comment:
Just a note: The description in the nuspec file does not need to be updated. This will be automatically handled when the package gets updated.

It is preferred that the nuspec do not get updated unless it is absolutely necessary.